db/1.9/modules/ROOT/pages/database-connector-troubleshooting.adoc

```diff
@@ -42,19 +78,58 @@ The following example shows how to use pooling profiles with the *Stored procedu
 	</flow>
 ----
 
-== Snowflake JDBC Driver Throws an Exception for a Stored Procedure
+
+== Troubleshoot Snowflake JDBC Driver That Throws an Exception for a Stored Procedure
 
 Some drivers have their own specifications when calling stored procedures. Running the application when you set the callable statement with curly brackets (`{` and `}`), for example, `{call doubleMyInt(:myInt)}`, causes the Snowflake JDBC driver to throw the following exception:
 
 `net.snowflake.client.jdbc.SnowflakeSQLException: Unsupported feature 'call'`
 
 
-=== Solution
+To resolve this issue:
 
 For Snowflake JDBC driver, set the callable statement without the curly brackets `{` and `}`, for example:
 
 `call doubleMyInt(:myInt)`
 
+
+== Understand Common Throws
+
+Here is a list of common throws messages and how to interpret them.
+
+* DB:BAD_SQL_SYNTAX
+
+ The provided SQL query contains invalid syntax.
+
+* DB:CANNOT_LOAD_DRIVER
+
+ Database Connector cannot load the JDBC driver.
+
+* DB:CANNOT_REACH
+
+ Database Connector cannot establish a connection with the RDBMS.
+
+* DB:CONNECTIVITY
+
+ Any of several possible connectivity issues, including poorly configured connections, lack of database response waiting results or fetching rows, loss of database connection, and so on.
```

Comment:
```suggestion
 Any of several possible connectivity issues, including poorly configured connections, lack of a database response while awaiting results or fetching rows, loss of database connection, and so on.
```

Reply:
? I'm not sure what this is meant to say: ...lack of database response waiting results or fetching rows,...